src/util.ts

```diff
@@ -249,3 +249,13 @@ function lastDotCmp(s: string): string {
 	const cmps = s.split('.');
 	return cmps[cmps.length - 1];
 }
+
+/**
+ * Strips file part (if any) from container name (last component of container path)
+ * For example TS may return the following name: /node_modules/vscode-jsonrpc/lib/cancellation.
+ * We consider that if name contains path separtor then container name is empty
+ * @param containerName
+ */
+function stripFileInfo(containerName: string): string {
```

Comment:
Could you explain why this is needed?
Iirc the `containerName` is used inside `SymbolDescriptor`, so changing it affects the uniqueness of it